src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -366,6 +372,29 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             };
         }
 
+        internal JObject GetHostJson()
+        {
+            var defaultJObject = new JObject();
+            var scriptPath = _applicationHostOptions.CurrentValue?.ScriptPath;
```

Comment:
Don't need the ? null check, since you're not handling the null if it happened on the next line anyways. Existing ReadDurableTaskConfig code doesn't check for null either.

Reply:
Thanks.  I change the logic to use the same as Durable.